migrations/migration_resource.py

```diff
@@ -0,0 +1,45 @@
+"""Migration Module
+
+Captures common methods needed by the system for migrations, by specializing
+the `fhirclient.Basic` class.
```

Comment:
Add something to explain how we use, i.e.:

A single `Basic` resource is maintained with the most recently (successfully) run migration revision held in the single `Basic.code` value.